src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -378,16 +378,15 @@ public function myIncludedInPriceTaxesShouldBe(int $taxTotal): void
     }
 
     /**
-     * @Then /^my (cart) should be empty$/
-     * @Then /^(cart) should be empty with no value$/
+     * @Then my cart should be empty
      */
-    public function myCartShouldBeEmpty(string $tokenValue): void
+    public function myCartShouldBeEmpty(): void
     {
-        $response = $this->shopClient->show(Resources::ORDERS, $tokenValue);
+        $tokenValue = $this->sharedStorage->get('cart_token');
```

Comment:
```suggestion
		Assert::true($this->sharedStorage->has('cart_token'));

        $tokenValue = $this->sharedStorage->get('cart_token');
```
Just to make sure, unless it can be null since the cart has been recreated, in that case check and ignore if missing.